euphonic_sqw_models/euphonic_wrapper.py

```diff
@@ -142,9 +161,9 @@ def _calculate_phonon_modes(self, qpts):
         if self.force_constants is None:
             raise RuntimeError('Force constants model not set')
         return self.force_constants.calculate_qpoint_phonon_modes(qpts,
-            weights=self.weights, asr=self.asr, dipole=self.dipole, eta_scale=self.eta_scale,
-            splitting=self.splitting, insert_gamma=self.insert_gamma, reduce_qpts=self.reduce_qpts,
-            use_c=self.use_c, n_threads=self.n_threads)
+            weights=self.weights, asr=self.asr, dipole=self.dipole, dipole_parameter=self.dipole_parameter,
+            eta_scale=self.eta_scale, splitting=self.splitting, insert_gamma=self.insert_gamma,
+            reduce_qpts=self.reduce_qpts, use_c=self.use_c, n_threads=self.n_threads)
```

Comment:
Noting that eta_scale continues to exist despite eta being deprecated. Is this just the deprecated option? Apparently yes looking at PR-170. Would you prefer to rename it to dipole_paramenter and then just pass dipole_parameter.

Reply:
Yes `eta_scale` here is the deprecated option. I wanted it to still be used so if someone uses `eta_scale` from `euphonic_sqw_models` (or `Horace-Euphonic-Interface`) they will get the deprecation message. If we just silently rename `eta_scale` -> `dipole_parameter` internally users won't know they're supposed to be using something else. I didn't want to have to manually raise another `DeprecationWarning` from here either, would rather let Euphonic do it.